lima.wdl

```diff
@@ -87,13 +87,13 @@ task Lima {
         ~{barcodeFile} \
         ~{basename(outputPrefix) + ".fl.bam"}
```

Comment:
Can't you use {outputPrefix} here directly instead of the basename ? Wouldn't that remove the necessity for copying?

Reply:
That would indeed be nicer. But for some reason the glob function in the outputs doesn't work when I try to glob(outputPrefix + "*.bam"). It doesn't find any files. So I need all files to be written to the working dir in order to use them downstream.

Reply:
The lima task gets one BAM file as input. And can output multiple de-multiplexed BAM files. So that's why I am using glob here.